### PR TITLE
start the move to sass

### PR DESCRIPTION
**Pull request by jmhauck**
@MikeTschudi this is the start of it...anything css is supposed to work just fine in scss so this is just a straight up move of the css to scss. Plan was to make sure this all went through just fine and then I'll start to make changes to leverage what we get from scss in a subsequent PR. Should be easier to look through and see the actual changes after this seemingly large pr.

### CODE REVIEW

File: src/assets/styles/styles.css [hunk 1]
```diff
@@ -13,15 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-@import "calcite-web.min.css";
-@import "esri-global-nav.css";
-
+@import url(c:\Users\john4818\Documents\GitHub\solutions-components\src\assets\styles\calcite-web.min.css);
```
Comment (MikeTschudi) [inline src/assets/styles/styles.css:16]: This absolute path will be a problem

Comment (jmhauck) [reply src/assets/styles/styles.css:16]: @MikeTschudi Thanks...digging around that but haven't seen how to handle yet.

Comment (jmhauck) [reply src/assets/styles/styles.css:16]: As far as I know...not supposed to muck with the css files directly...has to be a way to handle it.

Comment (MikeTschudi) [reply src/assets/styles/styles.css:16]: perhaps
```css
@import "calcite-web.min";
@import "esri-global-nav";
```
?  Extensions may be reason that the absolute path & the `@import url()` format are generated.

**Review by MikeTschudi — APPROVED** [review]
Nice!

**Comment by jmhauck** [comment]
@MikeTschudi after looking closer at calcite I see that we can lean on stencil a bit more and keep things a bit less cluttered :)

**Pull request merged**